ios-trivia-game/ViewControllers/HomeViewController.swift

```diff
@@ -70,7 +70,7 @@ class HomeViewController: UIViewController {
         FirebaseClient.instance.getGameRooms(complete: { (dictionary) in
             for (roomId, roomInfo) in dictionary {
                 let gameRoom = GameRoom(dictionary: roomInfo as! NSDictionary)
-                if gameRoom.state != GameRoom.State.end {
+                if gameRoom.state != GameRoom.State.end && Date().timeIntervalSince(gameRoom.created_time) <= Double(Constants.REFRESH_GAME_ROOM_INTERVAL){
```

Comment:
@zhiachong , @wassupnari we aren't clearing out stale game rooms.  What do you think of this check?